app/modules/filevault_escrow/filevault_escrow_model.php

```diff
@@ -19,7 +19,12 @@ public function __construct($serial = '')
         $this->rs['HddSerial'] = '';
 
         // Schema version, increment when creating a db migration
-        $this->schema_version = 0;
+        $this->schema_version = 1;
```

Comment:
If you want to improve the file vault_escrow module, please create another pull request